server/internal/core/domain/tx_request.go

```diff
@@ -91,3 +91,29 @@ type Receiver struct {
 func (r Receiver) IsOnchain() bool {
 	return len(r.OnchainAddress) > 0
 }
+
+type TxRequests []TxRequest
+
+func (t TxRequests) CountSpentVtxos() int {
```

Comment:
worth commenting why we skip Note and Swept vtxo maybe ?

Reply:
done